From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenny Root <kroot@google.com>
Date: Fri, 24 Jan 2020 14:08:53 -0800
Subject: ANDROID: gki_defconfig: enable NVDIMM/PMEM options

Options needed for the virtual platform and physical platforms via DT
to define PMEM regions for resume-on-reboot feature

Bug: 146400078
Change-Id: Icaffbd3a7425ac3d9914378fb356d32f9393eec0
Signed-off-by: Kenny Root <kroot@google.com>
[joneslee: Removed defconfig elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 cuttlefish.fragment | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cuttlefish.fragment b/cuttlefish.fragment
--- a/cuttlefish.fragment
+++ b/cuttlefish.fragment
@@ -15,5 +15,6 @@ CONFIG_SND_INTEL8X0=m
 CONFIG_USB_DUMMY_HCD=m
 CONFIG_RTC_DRV_TEST=m
 CONFIG_VIRTIO_PCI=m
+CONFIG_VIRTIO_PMEM=m
 CONFIG_VIRTIO_INPUT=m
 CONFIG_VIRTIO_MMIO=m
